src/Extension/SEODataExtension.php

```diff
@@ -180,11 +180,14 @@ public function MetaTagCollection($raw = false)
 		    if (!$raw) {
                 $metaTitle = MetaTitleTemplate::parse_meta_title($this->owner, $metaTitle);
             }
-		    $tags['title'] = $raw ? $metaTitle : HTML::createTag('title', [], $metaTitle);
-            $tags['meta_title'] = $raw ? $metaTitle : HTML::createTag('meta', array(
-                'name' => 'title',
-                'content' => $metaTitle,
-            ));
+            $tags['title'] = $raw ? $metaTitle : HTML::createTag('title', [], $metaTitle);
+
+            if ($includeMetaTitle && strtolower($includeMetaTitle) != 'false') {
```

Comment:
Why are you checking a boolean against a string here? that doesn't make any sense.